src/components/v5/shared/SearchSelect/SearchSelect.tsx

```diff
@@ -153,6 +155,19 @@ const SearchSelect = React.forwardRef<HTMLDivElement, SearchSelectProps>(
                 )
               ) : (
                 <>
+                  {showSearchValueAsOption && (
+                    <button
+                      type="button"
+                      className="text-sm md:hover:text-blue-400 flex items-center gap-2 min-h-[3.125rem]
+                      "
+                      onClick={() => onSelect?.(searchValue)}
+                    >
+                      <Avatar />
+                      <span className="max-w-[15.625rem] truncate">
+                        {searchValue}
+                      </span>
+                    </button>
+                  )}
```

Comment:
How does this work now? This will be shown when no option is found for the query but when the search value is not an address it will do nothing?

Reply:
What's the point of allowing invalid values in this input? I think we need to discuss this with the team